src/renderers/dom/client/eventPlugins/SelectEventPlugin.js

```diff
@@ -189,12 +196,18 @@ var SelectEventPlugin = {
         return constructSelectEvent(nativeEvent, nativeEventTarget);
 
       // Chrome and IE fire non-standard event when selection is changed (and
-      // sometimes when it hasn't).
+      // sometimes when it hasn't). IE's event fires out of order with respect
+      // to key and input events on deletion, so we discard it.
+      //
       // Firefox doesn't support selectionchange, so check selection status
       // after each key entry. The selection changes after keydown and before
       // keyup, but we check on keydown as well in the case of holding down a
       // key, when multiple keydown events are fired but only one keyup is.
+      // This is also our approach for IE handling, for the reason above.
       case topLevelTypes.topSelectionChange:
+        if (skipSelectionChangeEvent) {
+          break;
+        }
```

Comment:
`// fall through` is [only supported in ESLint 1.0.0](https://github.com/eslint/eslint/commit/ff0ad4dffc1ac7924154bc7618c362526992fadd). I think we're on an older version, so in that case, you need to pluralize `fall` to `// falls through`.

Also, make sure it's the last thing in the case, after the conditional.